x-pack/plugins/siem/scripts/gql_gen_client.json

```diff
@@ -14,7 +14,8 @@
       "ToStringArray": "string[]",
       "ToNumberArray": "number[]",
       "ToDateArray": "string[]",
-      "ToBooleanArray": "boolean[]"
+      "ToBooleanArray": "boolean[]",
+      "Date": "string"
```

Comment:
This is absolutely fantastic! Thank you _SO_ much for taking the time to make these improvements! 👏👏👏👏